source/solvers/vof.cc

```diff
@@ -639,11 +639,11 @@ VOF<dim>::setup_dofs()
   // multiphysics interface
   multiphysics->set_dof_handler(PhysicsID::free_surface, &dof_handler);
   multiphysics->set_solution(PhysicsID::free_surface, &present_solution);
-  // the fluid at present iteration is solved before the free surface, and
-  // after percolate is called for the previous iteration, so at the time the
-  // getter is called solution_m2 = solution_m1
-  // TODO deactivated for now (inertia is considered with a constant density),
-  // see if needed / to be debugged
+  // the fluid at present iteration is solved AFTER the free surface (see map
+  // solve_pre_fluid defined in multiphysics_interface.h), and after percolate
+  // is called for the previous iteration.
+  // NB: for now, inertia in fluid dynamics is considered with a constant
+  // density (see if needed / to be debugged)
```

Comment:
This is not true right now because all flags are set to false.
We'll have to investigate which is better once we have more difficult test cases.
```suggestion
  // the fluid at present iteration is solved BEFORE the free surface (see map
  // solve_pre_fluid defined in multiphysics_interface.h), and after percolate
  // is called for the previous iteration.
  // NB: for now, inertia in fluid dynamics is considered with a constant
  // density (see if needed / to be debugged)
```